src/components/account/CreateAccount.tsx

```diff
@@ -0,0 +1,156 @@
+          onChange={(e) =>
+            setFormInput({ ...formInput, lastName: e.target.value })
+          }
+        />
+
+        <label>Email address</label>
+        <input
+          type="text"
+          value={formInput.email}
+          placeholder="janejo@mailator.com"
+          onChange={(e) =>
+            setFormInput({ ...formInput, email: e.target.value })
+          }
+        />
+
+        <label>Amount</label>
+        <input
+          type="text"
+          value={formInput.amount}
+          placeholder="400"
+          onChange={(e) =>
+            setFormInput({ ...formInput, amount: e.target.value })
+          }
+        />
+
+        <label>Currency</label>
+        <input
+          type="text"
+          value={formInput.currency}
+          placeholder="USD"
+          onChange={(e) =>
+            setFormInput({ ...formInput, currency: e.target.value })
+          }
+        />
+
+        <label>Note</label>
+        <input
+          type="text"
+          value={formInput.note}
+          placeholder="a customer"
+          onChange={(e) => setFormInput({ ...formInput, note: e.target.value })}
+        />
+      </form>
+
+      <button
+        type="submit"
+        className="createAccountBtn"
+        onClick={() => {
+          const validateForm = validateCreateAccount(formInput)
+          const validationErrors = Object.values(validateForm).filter(
+            (error) => error !== ''
+          )
+          if (validationErrors.length !== 0) {
+            setFormValidationErrors(validationErrors)
+          } else {
+            setFormValidationErrors(null)
+            setRequestStatus('fetching')
+            fetch('https://nestjs-bank-app.herokuapp.com/accounts', {
+              method: 'POST',
+              headers: { 'Content-Type': 'application/json' },
+              body: JSON.stringify({
+                given_name: formInput.firstName,
+                family_name: formInput.lastName,
+                email_address: formInput.email,
+                id: formInput.id,
+                balance: {
+                  amount: parseInt(formInput.amount),
+                  currency: formInput.currency,
+                },
+                note: formInput.note,
+              }),
+            })
+              .then((res) => {
+                setFormValidationErrors(null)
+                if (!res.ok) {
+                  setRequestStatus('error')
+                  throw Error(res.statusText)
+                }
+                setFormInput(formInitialState)
+                setMessage('Account successfully created!')
+                setRequestStatus('success')
+              })
+              .catch((e) => {
+                setMessage(e.message)
+              })
+          }
+        }}
+      >
+        Create Account
+      </button>
+
+      <div className="requestStatus">
+        <RequestStatus
+          request={{ status: requestStatus, message: message }}
```

Comment:
just a shorthand when defining an object property if key and value have identical names
```suggestion
          request={{ status: requestStatus, message }}
```

Although I do like in other components where you rename this useState to errorMessage too
```suggestion
          request={{ status: requestStatus, message: errorMessage }}
```